cmake/extension.cmake

```diff
@@ -249,4 +249,11 @@ if(${HTTPPROXY} STREQUAL true)
   client_sdk_compile_definitions(-DHTTPPROXY)
 endif()
 
+if(${TPM2_TCTI_TYPE} MATCHES tpmrm0)
```

Comment:
since this is only applicable for DA=tpm20_ecdsa256. 
it would be better to keep this inside an "if" check for DA.

Reply:
Done.